packages/popper/src/utils/modifiers.ts

```diff
@@ -0,0 +1,36 @@
+import { mergeDeep } from 'utilitify';
+import { Modifiers } from 'popper.js';
+
+
+const getDefaultModifiers = (): Modifiers => ({
+  applyStyle: { enabled: false },
+});
+
+const getAutoFlipModifiers = (autoFlip: boolean): Modifiers => {
```

Comment:
The method will be better for reading if you write like this:
```
const getAutoFlipModifiers = (autoFlip: boolean): Modifiers => {
  return {
    preventOverflow: {
      enabled: autoFlip,
    },
    flip: {
      enabled: autoFlip,
      padding: 0,
    },
    hide: {
      enabled: autoFlip,
    },
  };
};
```

Why? Because you're violating the *DRY* principle. You have written two similar objects

Reply:
@xcritical-software/frontend what do you think?

Reply:
@tundraray I think this option more obvious. For a specific autoFlip state, we set only those settings that are needed. But if we assume that the default settings could change, then your option will be more correct.

Reply:
I would still add a link to the document, so as not to waste time searching for information what it is.